spec/components/schemas/events/message-status-event.ts

```diff
@@ -22,14 +22,43 @@ const messageEvent: SchemaObject = {
       },
       messageId: {
         title: 'Message ID',
-        description: 'ID of the message whose status is being delivered',
+        description: 'ID of the message whose status is being delivered (deprecated)',
```

Comment:
https://spec.openapis.org/oas/latest.html#fixed-fields-7

```suggestion
        description: 'ID of the message whose status is being delivered',
        deprecated: true,
```